estimator/lwe_bkw.py

```diff
@@ -2,7 +2,18 @@
 """
 See :ref:`Coded-BKW for LWE` for what is available.
 """
-from sage.all import ZZ, ceil, log, floor, sqrt, var, find_root, erf, oo, cached_function
+from sage.rings.all import QQ
+from sage.calculus.var import var
+from sage.functions.all import log
```

Comment:
I think lines 6 and 7 can be removed (they were picked up as unused imports by the CI here: https://github.com/malb/lattice-estimator/pull/77/checks) :-).